staging/src/k8s.io/cli-runtime/pkg/genericclioptions/config_flags_fake.go

```diff
@@ -96,7 +96,7 @@ type namespacedClientConfig struct {
 }
 
 func (c *namespacedClientConfig) Namespace() (string, bool, error) {
-	return c.namespace, false, nil
+	return c.namespace, true, nil
```

Comment:
I thought we agreed this to be:
```go
return c.namespace, len(c.namespace) > 0, nil
```
didn't we?

Reply:
You're right.